tox.ini

```diff
@@ -7,10 +6,11 @@ envlist=
   py35
 
 [testenv]
-deps=
-  -rrequirements.dev.txt
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps= -rtests_require
```

Comment:
I think it is good to be able to pip install the test reqs (see make install). As for having it in setup.py, it is friendly for anyone who uses python setup.py test